portfolio/src/main/webapp/style.css

```diff
@@ -2,24 +2,120 @@ body {
     font-size: 15px;
     font-family: "Comic Sans MS", cursive, sans-serif;
     background: url("./img/congruent_pentagon.png"), #6F4A38;
-    background-repeat: repeat;
+    /* background-repeat: repeat; */
```

Comment:
nit: Remove this line if not needed, or maybe add a comment like "FIXME: add back when ..."

Reply:
Ack